src/infrastructure/utils/note.ts

```diff
@@ -0,0 +1,34 @@
+import { type OutputData } from '@editorjs/editorjs';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+/**
+ * Get the title of the note
+ *
+ * @param content - content of the note
+ * @returns the title of the note
+ */
+export const getTitle = (content: OutputData): string => {
```

Comment:
please, use named function instead of anonymous. And it's better to name it `composeNoteTitle` and make it accept Note as argument — that will make this method more clean and useful. Right not it depends on Editor.js, but it's better to depend on our entity — Note

```suggestion
export function composeNoteTitle(note: Note): string {
```